### PR TITLE
Adding token: BMOS from chain Terra

### PR DESCRIPTION
**Pull request by bitmos-io**
## Description
Adding token: BMOS from chain Terra

### Adding Assets
If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [x] Ready -- A liquidity pool has been created. The pool ID is: ______
- [ ] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

**Comment by bitmos-io** [comment]
Add Bitmos token BMOS

**Comment by JeremyParish69** [comment]
Is there someone with a longer github history who can leave a thumbs up on this PR?

**Comment by JeremyParish69** [comment]
Please have someone with a longer GitHub history leave a comment on this PR.

**Comment by JeremyParish69** [comment]
And let us know the ID of the pool with liquidity.

**Comment by ramenpepe** [comment]
looks like they are here

https://t.me/bitmos_community
https://app.astroport.fi/pools/terra1er4zgsgv6uj5gqw8y3630pyfar4ctgmetv34ak90n0adqcpv0gwsxlj37l

**Comment by dpny518** [comment]
I know this project, met with the team IRL at cosmos event

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -13695,6 +13695,47 @@
         "osmosis-price:ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4:1474",
         "peg:collateralized"
       ]
+    },
+     {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:13699]: Please do not update this generated assetlist file. Only the zone_assets file.

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -1,2687 +1,2693 @@
...
-      "osmosis_verified": true,
-      "transfer_methods": [
-        {
-          "name": "Axelar Bridge",
-          "type": "integrated_bridge",
-          "counterparty": [
-            {
-              "chain_name": "ethereum",
-              "unwrappedAssetId": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"
+    "$schema": "../zone.schema.json",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2]: Please do not reformat the zone_assets file with the 4-space-wide tabs.

**Comment by abhishekhan** [comment]
👍

**Review by bitmos-io — COMMENTED** [review]
revision changes made as per advised

**Pull request merged**